ui/plugins/controller.js

```diff
@@ -330,6 +330,7 @@ treeherder.controller('PluginCtrl', [
                 ThJobModel.retrigger($scope.repoName, job_id_list).then(function() {
                     // XXX: Remove this after 1134929 is resolved.
                     return ThJobDetailModel.getJobDetails({"title": "buildbot_request_id",
+                                                           "repository": $scope.repoName,
                                                            "job_id__in": job_id_list.join(',')})
```

Comment:
I'm not sure why I put the parameters in quotes initially, as it isn't required.